pages/common/id3tag.md

```diff
@@ -0,0 +1,11 @@
+# id3tag
+
+> Tool for reading, writing, and manipulating ID3v1 and ID3v2 tags.
+
+- Set artist and title tag of an MP3:
```

Comment:
of an MP3 -> of an MP3 file.

Also, you mention artist and title tag in the description, but the command also sets the album. If you want, we can add another example for album or just modify this description to add album too.